tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -1393,13 +1393,13 @@ Status Converter::BuildCudaEngine(
   TF_RETURN_IF_ERROR(TrtPrecisionModeToName(
       precision_mode_, &precision_mode_str));
   string trt_version_str = GetLoadedTensorRTVersion();
-  string trt_network_name =
-      "TF" + string(TF_VERSION_STRING) + "-" +
-      "TRT" + trt_version_str + "-" +
-      "Precision-" + precision_mode_str + "-" +
-      "Calibration-" + std::to_string(use_calibration_) + "-" +
-      "Max-Batch-Size-" + std::to_string(max_batch_size) + "-" +
-      "Max-Workspace-Size-" + std::to_string(max_workspace_size_bytes);
+  string trt_network_name = StrCat(
+      "TF:", string(TF_VERSION_STRING), ", ",
```

Comment:
nit: please remove calls to string() and to_string(). There is no need to construct a string explicitly with StrCat.

Reply:
Done